src/Mutator/Util/AbstractIdenticalComparison.php

```diff
@@ -226,18 +235,66 @@ private function getStaticMethodReturnType(Node\Name $class, Node\Identifier $me
         }
     }
 
-    private function getClassConstantValue(Node\Name $class, Node\Identifier $name): mixed
+    private function getGlobalConstantType(Node\Name $name): ?string
+    {
+        if (PHP_VERSION_ID < self::REFLECTION_CONSTANT_MIN_VERSION) {
+            return null;
+        }
+
+        try {
+            // @phpstan-ignore class.notFound
+            $reflection = new ReflectionConstant($name->toString());
```

Comment:
PHPStorm reports a false-positive at this line.
[Upstream bug](https://youtrack.jetbrains.com/issue/WI-81604/ReflectionConstant-is-available-starting-with-8.4-PHP-version)